app/src/main/java/ak/www/kursakademiaandroida/core/di/NetworkModule.kt

```diff
@@ -0,0 +1,37 @@
+package ak.www.kursakademiaandroida.core.di
+
+import ak.www.kursakademiaandroida.BuildConfig
+import ak.www.kursakademiaandroida.features.data.RickAndMortyApi
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import org.koin.dsl.module
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+val networkModule = module {
+
+    single {
+        HttpLoggingInterceptor()
+            .apply {
+                if (BuildConfig.DEBUG) HttpLoggingInterceptor.Level.BODY
```

Comment:
Wynik z bloku if else powinien być przypisany do zmiennej `level`. Umknęło mi to na wideo, ale już dodałem komentarz w tej sprawie pod nagraniem.